### PR TITLE
Fix status handling of sourcing projects.

### PR DESCRIPTION
**Pull request by eodolphi**
- Send email when task is closed
- Set project to realised when all tasks are realised
- Send survey link to project initiator
- Send email (with survey link) to project organization

BB-7956 #resolve
BB-7980 #resolve
BB-7995 #resolve
BB-8024 #resolve

### CODE REVIEW

File: bluebottle/projects/models.py [hunk 1]
```diff
@@ -542,6 +542,12 @@ def status_changed(self, old_status, new_status):
 
             bb_track("Project Completed", data)
 
+    def check_task_status(self):
+        if ((not self.is_funding) and
```
Comment (gannetson) [inline bluebottle/projects/models.py]: These brackets aren't needed, right?

File: bluebottle/projects/templates/projects/mails/organization_project_complete.mail.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+{% block content %}
+    {% blocktrans %}
+
+        Hi {{ receiver_name }},
+        <br><br>
+        Congratulations! Your project "{{ title }}" has been realised! It's time
+        to celebrate!<br>
+        <br>
+        TODO: fix content
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/organization_project_complete.mail.html]: Sure?! ^^

File: bluebottle/projects/templates/projects/mails/organization_project_complete.mail.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{% extends "base.mail.txt" %}
+{% load i18n %}
+
+{% block content %}
+{% blocktrans with receiver_name=to.short_name  %}
+
+Hi {{receiver_name}},
+
+TODO: fix content
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/organization_project_complete.mail.txt]: Same ^^

File: bluebottle/projects/templates/projects/mails/project_complete.mail.html [hunk 4]
```diff
@@ -5,12 +5,14 @@
     {% blocktrans %}
 
         Hi {{ receiver_name }},
-        <br><br>
-        Congratulations! Your project "{{ title }}" has been realised! It's time
-        to celebrate!<br>
-        <br>
-        Why don't you share this success with your supporters.<br>
+        TODO: Fix content
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/project_complete.mail.html]: Too ^^

File: bluebottle/projects/tests/test_mail.py [hunk 5]
```diff
@@ -19,14 +21,34 @@ def setUp(self):
...
         self.project.save()
 
         self.assertEquals(len(mail.outbox), 1)
-        self.assertNotEquals(mail.outbox[0].subject.find("has been realised"), -1)
+        email = mail.outbox[0]
+
+        self.assertNotEquals(email.subject.find("has been realised"), -1)
+        self.assertTrue(self.survey.url(self.project, user_type='initiator') in email.body)
+
+    def test_complete_with_organisation(self):
```
Comment (gannetson) [inline bluebottle/projects/tests/test_mail.py]: US of A spelling please ^^ zzzz

File: bluebottle/tasks/templates/tasks/mails/task_status_closed.mail.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{% extends "base.mail.html" %}
+{% load i18n %}
+
+{% block content %}
+    {% blocktrans with receiver_name=to.short_name %}
+
+        Hello {{ receiver_name }}
+        TODO: Fix content
```
Comment (gannetson) [inline bluebottle/tasks/templates/tasks/mails/task_status_closed.mail.html]: Guess we just have to create a ticket for this. Make sure we add all the mails that need content.

File: bluebottle/projects/models.py (hunk 1 above)
Comment (eodolphi) [inline bluebottle/projects/models.py]: I tend to add brackets here even tough I know not binds stronger than and. But yeah, they can be removed.

File: bluebottle/tasks/templates/tasks/mails/task_status_closed.mail.html (hunk 6 above)
Comment (eodolphi) [inline bluebottle/tasks/templates/tasks/mails/task_status_closed.mail.html]: I created a ticket now, and assigned it to Floortje.

File: bluebottle/projects/templates/projects/mails/organization_project_complete.mail.html [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    {% endblocktrans %}
+
+    {% if survey_link %}
+        {% blocktrans with link=survey_link%}
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/organization_project_complete.mail.html]: pep8

File: bluebottle/projects/templates/projects/mails/project_complete.mail.html [hunk 8]
```diff
@@ -5,12 +5,14 @@
...
     {% endblocktrans %}
+
+    {% if survey_link %}
+        {% blocktrans with link=survey_link%}
+        <a href="{{ link}}">Please take a moment to fill in a survey about the impact you created<a/>
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/project_complete.mail.html]: template tag

File: bluebottle/projects/templates/projects/mails/project_complete.mail.txt [hunk 9]
```diff
@@ -11,6 +11,12 @@ celebrate!
 
 Why don't you share this success with your supporters.
 {% endblocktrans %}
+{% if survey_link %}
+{% blocktrans with link=survey_link%}
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/project_complete.mail.txt]: here

File: bluebottle/projects/templates/projects/mails/project_complete.mail.txt [hunk 10]
```diff
@@ -11,6 +11,12 @@ celebrate!
...
+Please take a moment to fill in a survey about the impact you created:
+{{link}}
```
Comment (gannetson) [inline bluebottle/projects/templates/projects/mails/project_complete.mail.txt]: here

**Pull request merged**